### PR TITLE
docs: add 4.0.0-rc.5 changelog

### PR DESCRIPTION
**Pull request by chenshuai2144**
## 4.0.0-rc.5

`2020-02-16`

- 🐞 Fix Form.Item validateFirst prevent finial submit. [#21329](https://github.com/ant-design/ant-design/pull/21329)
- 🐞 Fixed cursor position issue when deleting consecutive identical numbers. [#21344](https://github.com/ant-design/ant-design/pull/21344)
- 💄 Remove excess background color. [#21365](https://github.com/ant-design/ant-design/pull/21365)
- 💄 Optimize the mouse style for the `disabled` state of the DatePicker component. [#21352](https://github.com/ant-design/ant-design/pull/21352)
- 🐞 Fix Affix throws `Cannot read property getBoundingClientRect` in mobile device. [#21350](https://github.com/ant-design/ant-design/pull/21350)
- 🐞 Fix incorrect label width when screen is less than xs. [#21222](https://github.com/ant-design/ant-design/pull/21222)
- 🐞 Fix Input size is large height style. [#21338](https://github.com/ant-design/ant-design/pull/21338)
- 🐞 Fix Badge `color` not working when contains children. [#21333](https://github.com/ant-design/ant-design/pull/21333)
- 🐞 Fix Alert close button extra padding. [#21325](https://github.com/ant-design/ant-design/pull/21325)
- 💄 Tweak Steps 1px align issue. [#21306](https://github.com/ant-design/ant-design/pull/21306)
- 💄 Fix legacy Button.Group `large` size style issue. [#21307](https://github.com/ant-design/ant-design/pull/21307)
- 💄 Fix Input border radius with suffix, style in Firefox, TextArea allowClear style issues. [#21316](https://github.com/ant-design/ant-design/pull/21316)
- 🐞 Pagination will pass `disabled` prop to prev/next buttons return by `itemRender`. [#21361](https://github.com/ant-design/ant-design/pull/21361)
- 🌎 Added Azerbaijani translation. [#21387](https://github.com/ant-design/ant-design/pull/21387)
- Typescript
  - 🔷 add MenuItemGroupProps export. [#21356](https://github.com/ant-design/ant-design/pull/21356)
  - 🔷 Table export `ColumnProps`. [#21321](https://github.com/ant-design/ant-design/pull/21321)
------
## 4.0.0-rc.5

`2020-02-16`

- 🐞 修复 Form.Item 设置 `validateFirst` 导致表单无法提交的问题。[#21329](https://github.com/ant-design/ant-design/pull/21329)
- 🐞 解决了删除连续相同数字时的光标位置问题。[#21344](https://github.com/ant-design/ant-design/pull/21344)
- 💄 移除掉多余的背景色。[#21365](https://github.com/ant-design/ant-design/pull/21365)
- 💄 优化 DatePicker 组件 `disabled` 状态的鼠标样式。[#21352](https://github.com/ant-design/ant-design/pull/21352)
- 🐞 修复 Affix 在移动设备下抛错 `Cannot read property getBoundingClientRect` 的问题。[#21350](https://github.com/ant-design/ant-design/pull/21350)
- 🐞 修复屏幕小于 xs 时标签宽度不正确的问题。[#21222](https://github.com/ant-design/ant-design/pull/21222)
- 🐞 修复 Input 在 `size` 为 `large` 时的高度问题。[#21338](https://github.com/ant-design/ant-design/pull/21338)
- 🐞 修复 Badge 包裹模式下 color 属性失效的问题。[#21333](https://github.com/ant-design/ant-design/pull/21333)
- 🐞 修复 Alert 关闭按钮额外的 padding。[#21325](https://github.com/ant-design/ant-design/pull/21325)
- 💄 微调 Steps 文本 1px 使其居中对齐。[#21306](https://github.com/ant-design/ant-design/pull/21306)
- 💄 修复遗留的 Button.Group `large` 尺寸的样式问题。[#21307](https://github.com/ant-design/ant-design/pull/21307)
- 💄 修正 suffix，Firefox 中的样式，TextArea allowClear 中的输入边框半径的样式问题。[#21316](https://github.com/ant-design/ant-design/pull/21316)
- 🐞 Pagination 自定义 `itemRender` 返回的上一页下一页现在会补充 `disabled` 属性。[#21361](https://github.com/ant-design/ant-design/pull/21361)
- 🌎 添加了阿塞拜疆语翻译。[#21387](https://github.com/ant-design/ant-design/pull/21387)
- Typescript
  - 🔷 添加`MenuItemGroupProps`导出。 [#21356](https://github.com/ant-design/ant-design/pull/21356)
  - 🔷 Table 导出`ColumnProps`。 [#21321](https://github.com/ant-design/ant-design/pull/21321)

-----
[View rendered CHANGELOG.en-US.md](https://github.com/ant-design/ant-design/blob/4.0.0-rc.5-changelog/CHANGELOG.en-US.md)
[View rendered CHANGELOG.zh-CN.md](https://github.com/ant-design/ant-design/blob/4.0.0-rc.5-changelog/CHANGELOG.zh-CN.md)

### CODE REVIEW

File: CHANGELOG.en-US.md [hunk 1]
```diff
@@ -15,6 +15,28 @@ timeline: true
 
 ---
 
+## 4.0.0-rc.5
+
+`2020-02-16`
+
+- 🐞 Fix Form.Item validateFirst prevent finial submit. [#21329](https://github.com/ant-design/ant-design/pull/21329)
```
Comment (afc163) [inline CHANGELOG.en-US.md]:
```suggestion
- 🐞 Fix Form.Item `validateFirst` prevent finial submit. [#21329](https://github.com/ant-design/ant-design/pull/21329)
```

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
+- 🐞 修复 Input 在 `size` 为 `large` 时的高度问题。[#21338](https://github.com/ant-design/ant-design/pull/21338)
+- 🐞 修复 Badge 包裹模式下 color 属性失效的问题。[#21333](https://github.com/ant-design/ant-design/pull/21333)
+- 🐞 修复 Alert 关闭按钮额外的 padding。[#21325](https://github.com/ant-design/ant-design/pull/21325)
+- 💄 微调 Steps 文本 1px 使其居中对齐。[#21306](https://github.com/ant-design/ant-design/pull/21306)
+- 💄 修复遗留的 Button.Group `large` 尺寸的样式问题。[#21307](https://github.com/ant-design/ant-design/pull/21307)
+- 💄 修正 suffix，Firefox 中的样式，TextArea allowClear 中的输入边框半径的样式问题。[#21316](https://github.com/ant-design/ant-design/pull/21316)
+- 🐞 Pagination 自定义 `itemRender` 返回的上一页下一页现在会补充 `disabled` 属性。[#21361](https://github.com/ant-design/ant-design/pull/21361)
+- 🌎 添加了阿塞拜疆语翻译。[#21387](https://github.com/ant-design/ant-design/pull/21387)
+- Typescript
+  - 🔷 添加`MenuItemGroupProps`导出。 [#21356](https://github.com/ant-design/ant-design/pull/21356)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: ` 前后少空格。

Comment (afc163) [reply CHANGELOG.zh-CN.md]: 。后多了空格。

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
+`2020-02-16`
+
+- 🐞 修复 Form.Item 设置 `validateFirst` 导致表单无法提交的问题。[#21329](https://github.com/ant-design/ant-design/pull/21329)
+- 🐞 解决了删除连续相同数字时的光标位置问题。[#21344](https://github.com/ant-design/ant-design/pull/21344)
+- 💄 移除掉多余的背景色。[#21365](https://github.com/ant-design/ant-design/pull/21365)
+- 💄 优化 DatePicker 组件 `disabled` 状态的鼠标样式。[#21352](https://github.com/ant-design/ant-design/pull/21352)
+- 🐞 修复 Affix 在移动设备下抛错 `Cannot read property getBoundingClientRect` 的问题。[#21350](https://github.com/ant-design/ant-design/pull/21350)
+- 🐞 修复屏幕小于 xs 时标签宽度不正确的问题。[#21222](https://github.com/ant-design/ant-design/pull/21222)
+- 🐞 修复 Input 在 `size` 为 `large` 时的高度问题。[#21338](https://github.com/ant-design/ant-design/pull/21338)
+- 🐞 修复 Badge 包裹模式下 color 属性失效的问题。[#21333](https://github.com/ant-design/ant-design/pull/21333)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]:
```suggestion
- 🐞 修复 Badge 包裹模式下 `color` 属性失效的问题。[#21333](https://github.com/ant-design/ant-design/pull/21333)
```

File: CHANGELOG.zh-CN.md [hunk 4]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
+- 🐞 修复 Alert 关闭按钮额外的 padding。[#21325](https://github.com/ant-design/ant-design/pull/21325)
+- 💄 微调 Steps 文本 1px 使其居中对齐。[#21306](https://github.com/ant-design/ant-design/pull/21306)
+- 💄 修复遗留的 Button.Group `large` 尺寸的样式问题。[#21307](https://github.com/ant-design/ant-design/pull/21307)
+- 💄 修正 suffix，Firefox 中的样式，TextArea allowClear 中的输入边框半径的样式问题。[#21316](https://github.com/ant-design/ant-design/pull/21316)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]:
```suggestion
- 💄 修正 TextArea `allowClear` 中的输入边框半径的样式问题和 Input `Suffix` 在 Firefox 下的样式问题。[#21316](https://github.com/ant-design/ant-design/pull/21316)
```

英文也更新一下。

File: CHANGELOG.en-US.md [hunk 5]
```diff
@@ -15,6 +15,28 @@ timeline: true
 
 ---
 
+## 4.0.0-rc.5
+
+`2020-02-16`
+
+- 🐞 Fix Form.Item `validateFirst` prevent finial submit. [#21329](https://github.com/ant-design/ant-design/pull/21329)
```
Comment (afc163) [inline CHANGELOG.en-US.md]: Fix 还是 Fixed 需要统一一下。

File: CHANGELOG.en-US.md [hunk 6]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
+- 🐞 Fix Input size is large height style. [#21338](https://github.com/ant-design/ant-design/pull/21338)
+- 🐞 Fix Badge `color` not working when contains children. [#21333](https://github.com/ant-design/ant-design/pull/21333)
+- 🐞 Fix Alert close button extra padding. [#21325](https://github.com/ant-design/ant-design/pull/21325)
+- 💄 Tweak Steps 1px align issue. [#21306](https://github.com/ant-design/ant-design/pull/21306)
+- 💄 Fix legacy Button.Group `large` size style issue. [#21307](https://github.com/ant-design/ant-design/pull/21307)
+- 💄 Fix Input border radius with suffix, style in Firefox, TextArea allowClear style issues. [#21316](https://github.com/ant-design/ant-design/pull/21316)
+- 🐞 Pagination will pass `disabled` prop to prev/next buttons return by `itemRender`. [#21361](https://github.com/ant-design/ant-design/pull/21361)
+- 🌎 Added Azerbaijani translation. [#21387](https://github.com/ant-design/ant-design/pull/21387) [@orkhan-huseyn](https://github.com/orkhan-huseyn)
+- Typescript
+  - 🔷 add MenuItemGroupProps export. [#21356](https://github.com/ant-design/ant-design/pull/21356)
```
Comment (afc163) [inline CHANGELOG.en-US.md]:
```suggestion
  - 🔷 export `MenuItemGroupProps`. [#21356](https://github.com/ant-design/ant-design/pull/21356)
```

File: CHANGELOG.en-US.md [hunk 7]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
+- 🐞 Fix incorrect label width when screen is less than xs. [#21222](https://github.com/ant-design/ant-design/pull/21222)
+- 🐞 Fix Input size is large height style. [#21338](https://github.com/ant-design/ant-design/pull/21338)
+- 🐞 Fix Badge `color` not working when contains children. [#21333](https://github.com/ant-design/ant-design/pull/21333)
+- 🐞 Fix Alert close button extra padding. [#21325](https://github.com/ant-design/ant-design/pull/21325)
+- 💄 Tweak Steps 1px align issue. [#21306](https://github.com/ant-design/ant-design/pull/21306)
+- 💄 Fix legacy Button.Group `large` size style issue. [#21307](https://github.com/ant-design/ant-design/pull/21307)
+- 💄 Fix Input border radius with suffix, style in Firefox, TextArea allowClear style issues. [#21316](https://github.com/ant-design/ant-design/pull/21316)
+- 🐞 Pagination will pass `disabled` prop to prev/next buttons return by `itemRender`. [#21361](https://github.com/ant-design/ant-design/pull/21361)
+- 🌎 Added Azerbaijani translation. [#21387](https://github.com/ant-design/ant-design/pull/21387) [@orkhan-huseyn](https://github.com/orkhan-huseyn)
```
Comment (afc163) [inline CHANGELOG.en-US.md]: 用 🇦🇿 做 emoji。

File: CHANGELOG.zh-CN.md [hunk 8]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
+
+- 🐞 修复 Form.Item 设置 `validateFirst` 导致表单无法提交的问题。[#21329](https://github.com/ant-design/ant-design/pull/21329)
+- 🐞 解决了删除连续相同数字时的光标位置问题。[#21344](https://github.com/ant-design/ant-design/pull/21344)
+- 💄 移除掉多余的背景色。[#21365](https://github.com/ant-design/ant-design/pull/21365)
+- 💄 优化 DatePicker 组件 `disabled` 状态的鼠标样式。[#21352](https://github.com/ant-design/ant-design/pull/21352)
+- 🐞 修复 Affix 在移动设备下抛错 `Cannot read property getBoundingClientRect` 的问题。[#21350](https://github.com/ant-design/ant-design/pull/21350)
+- 🐞 修复屏幕小于 xs 时标签宽度不正确的问题。[#21222](https://github.com/ant-design/ant-design/pull/21222)
+- 🐞 修复 Input 在 `size` 为 `large` 时的高度问题。[#21338](https://github.com/ant-design/ant-design/pull/21338)
+- 🐞 修复 Badge 包裹模式下 color 属性失效的问题。[#21333](https://github.com/ant-design/ant-design/pull/21333)
+- 🐞 修复 Alert 关闭按钮额外的 padding。[#21325](https://github.com/ant-design/ant-design/pull/21325)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]:
```suggestion
- 🐞 修复 Alert 关闭按钮额外的 `padding`。[#21325](https://github.com/ant-design/ant-design/pull/21325)
```

File: CHANGELOG.zh-CN.md [hunk 9]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
+- 💄 微调 Steps 文本 1px 使其居中对齐。[#21306](https://github.com/ant-design/ant-design/pull/21306)
+- 💄 修复遗留的 Button.Group `large` 尺寸的样式问题。[#21307](https://github.com/ant-design/ant-design/pull/21307)
+- 💄 修正 suffix，Firefox 中的样式，TextArea allowClear 中的输入边框半径的样式问题。[#21316](https://github.com/ant-design/ant-design/pull/21316)
+- 🐞 Pagination 自定义 `itemRender` 返回的上一页下一页现在会补充 `disabled` 属性。[#21361](https://github.com/ant-design/ant-design/pull/21361)
+- 🌎 添加了阿塞拜疆语翻译。[#21387](https://github.com/ant-design/ant-design/pull/21387) [@orkhan-huseyn](https://github.com/orkhan-huseyn)
+- Typescript
+  - 🔷 添加`MenuItemGroupProps`导出。 [#21356](https://github.com/ant-design/ant-design/pull/21356)
+  - 🔷 Table 导出`ColumnProps`。 [#21321](https://github.com/ant-design/ant-design/pull/21321)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]:
```suggestion
  - 🔷 Table 导出 `ColumnProps`。 [#21321](https://github.com/ant-design/ant-design/pull/21321)
```

File: CHANGELOG.zh-CN.md [hunk 10]
```diff
@@ -15,6 +15,28 @@ timeline: true
 
 ---
 
+## 4.0.0-rc.5
+
+`2020-02-16`
+
+- 🐞 修复 Form.Item 设置 `validateFirst` 导致表单无法提交的问题。[#21329](https://github.com/ant-design/ant-design/pull/21329)
+- 🐞 解决了删除连续相同数字时的光标位置问题。[#21344](https://github.com/ant-design/ant-design/pull/21344)
+- 💄 移除掉多余的背景色。[#21365](https://github.com/ant-design/ant-design/pull/21365)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: 没头没尾，这是针对什么组件的改动？

File: CHANGELOG.zh-CN.md [hunk 11]
```diff
@@ -15,6 +15,28 @@ timeline: true
 
 ---
 
+## 4.0.0-rc.5
+
+`2020-02-16`
+
+- 🐞 修复 Form.Item 设置 `validateFirst` 导致表单无法提交的问题。[#21329](https://github.com/ant-design/ant-design/pull/21329)
+- 🐞 解决了删除连续相同数字时的光标位置问题。[#21344](https://github.com/ant-design/ant-design/pull/21344)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]: 没头没尾，这是针对什么组件的改动？

File: CHANGELOG.en-US.md (hunk 5 above)
Comment (chenshuai2144) [reply CHANGELOG.en-US.md]: 别人都是 fix，我也改一成 fix 吧

File: CHANGELOG.en-US.md [hunk 12]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
+## 4.0.0-rc.5
+
+`2020-02-16`
+
+- 🐞 Fix Form.Item `validateFirst` prevent finial submit. [#21329](https://github.com/ant-design/ant-design/pull/21329)
+- 🐞 Fixed cursor position issue when deleting consecutive identical numbers. [#21344](https://github.com/ant-design/ant-design/pull/21344)
+- 💄 Remove excess background color. [#21365](https://github.com/ant-design/ant-design/pull/21365)
+- 💄 Optimize the mouse style for the `disabled` state of the DatePicker component. [#21352](https://github.com/ant-design/ant-design/pull/21352)
+- 🐞 Fix Affix throws `Cannot read property getBoundingClientRect` in mobile device. [#21350](https://github.com/ant-design/ant-design/pull/21350)
+- 🐞 Fix incorrect label width when screen is less than xs. [#21222](https://github.com/ant-design/ant-design/pull/21222)
```
Comment (afc163) [inline CHANGELOG.en-US.md]: 没写是什么组件的改动。

File: CHANGELOG.en-US.md (hunk 7 above)
Comment (chenshuai2144) [reply CHANGELOG.en-US.md]: 这个 emoj i windows 上看起来就像乱码

**Comment by chenshuai2144** [comment]
自从回了家，我就没见过你们的头像长啥样

**Comment by yoyo837** [comment]
> 自从回了家，我就没见过你们的头像长啥样

我绑了host 看得见 😄

File: CHANGELOG.en-US.md (hunk 7 above)
Comment (afc163) [reply CHANGELOG.en-US.md]: 你找找 windows 上的阿塞拜疆国旗 emoji

File: CHANGELOG.en-US.md [hunk 13]
```diff
@@ -15,6 +15,28 @@ timeline: true
 
 ---
 
+## 4.0.0-rc.5
+
+`2020-02-16`
+
+- 🐞 Fix Form.Item `validateFirst` prevent finial submit. [#21329](https://github.com/ant-design/ant-design/pull/21329)
+- 🐞 Fix InputNumber cursor position issue when deleting consecutive identical numbers. [#21344](https://github.com/ant-design/ant-design/pull/21344)
+- 💄 Menu remove excess background color. [#21365](https://github.com/ant-design/ant-design/pull/21365)
```
Comment (afc163) [inline CHANGELOG.en-US.md]: Remove Menu

File: CHANGELOG.en-US.md [hunk 14]
```diff
@@ -15,6 +15,28 @@ timeline: true
 
 ---
 
+## 4.0.0-rc.5
+
+`2020-02-16`
+
+- 🐞 Fix Form.Item `validateFirst` prevent finial submit. [#21329](https://github.com/ant-design/ant-design/pull/21329)
+- 🐞 Fix InputNumber cursor position issue when deleting consecutive identical numbers. [#21344](https://github.com/ant-design/ant-design/pull/21344)
+- 💄 remove Menu excess background color. [#21365](https://github.com/ant-design/ant-design/pull/21365)
```
Comment (afc163) [inline CHANGELOG.en-US.md]:
```suggestion
- 💄 Remove Menu excess background color. [#21365](https://github.com/ant-design/ant-design/pull/21365)
```

File: CHANGELOG.zh-CN.md [hunk 15]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
+## 4.0.0-rc.5
+
+`2020-02-16`
+
+- 🐞 修复 Form.Item 设置 `validateFirst` 导致表单无法提交的问题。[#21329](https://github.com/ant-design/ant-design/pull/21329)
+- 🐞 解决了 InputNumber 删除连续相同数字时的光标位置问题。[#21344](https://github.com/ant-design/ant-design/pull/21344)
+- 💄 Menu 移除掉多余的背景色。[#21365](https://github.com/ant-design/ant-design/pull/21365)
+- 💄 优化 DatePicker 组件 `disabled` 状态的鼠标样式。[#21352](https://github.com/ant-design/ant-design/pull/21352)
+- 🐞 修复 Affix 在移动设备下抛错 `Cannot read property getBoundingClientRect` 的问题。[#21350](https://github.com/ant-design/ant-design/pull/21350)
+- 🐞 修复 Form 在屏幕小于 xs 时标签宽度不正确的问题。[#21222](https://github.com/ant-design/ant-design/pull/21222)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]:
```suggestion
- 🐞 修复 Form 在屏幕小于 `xs` 时标签宽度不正确的问题。[#21222](https://github.com/ant-design/ant-design/pull/21222)
```

File: CHANGELOG.en-US.md (hunk 5 above)
Comment (afc163) [inline CHANGELOG.en-US.md]:
```suggestion
- 🐞 Fix Form.Item `validateFirst` prevent form submit. [#21329](https://github.com/ant-design/ant-design/pull/21329)
```

File: CHANGELOG.en-US.md [hunk 16]
```diff
@@ -15,6 +15,28 @@ timeline: true
...
+- 🐞 Fix Input `size` is `large` height style. [#21338](https://github.com/ant-design/ant-design/pull/21338)
+- 🐞 Fix Badge `color` not working when contains children. [#21333](https://github.com/ant-design/ant-design/pull/21333)
+- 🐞 Fix Alert close button extra padding. [#21325](https://github.com/ant-design/ant-design/pull/21325)
+- 💄 Tweak Steps 1px align issue. [#21306](https://github.com/ant-design/ant-design/pull/21306)
+- 💄 Fix legacy Button.Group `large` size style issue. [#21307](https://github.com/ant-design/ant-design/pull/21307)
+- 💄 Fix Input border radius with suffix, style in Firefox, TextArea allowClear style issues. [#21316](https://github.com/ant-design/ant-design/pull/21316)
+- 🐞 Pagination will pass `disabled` prop to prev/next buttons return by `itemRender`. [#21361](https://github.com/ant-design/ant-design/pull/21361)
+- 🇦🇿 Added Azerbaijani translation. [#21387](https://github.com/ant-design/ant-design/pull/21387) [@orkhan-huseyn](https://github.com/orkhan-huseyn)
+- Typescript
+  - 🔷 Export `MenuItemGroupProps` export. [#21356](https://github.com/ant-design/ant-design/pull/21356)
```
Comment (afc163) [inline CHANGELOG.en-US.md]:
```suggestion
  - 🔷 Menu export `MenuItemGroupProps`. [#21356](https://github.com/ant-design/ant-design/pull/21356)
```

File: CHANGELOG.zh-CN.md [hunk 17]
```diff
@@ -34,7 +34,7 @@ timeline: true
 - 🐞 Pagination 自定义 `itemRender` 返回的上一页下一页现在会补充 `disabled` 属性。[#21361](https://github.com/ant-design/ant-design/pull/21361)
 - 🇦🇿 添加了阿塞拜疆语翻译。[#21387](https://github.com/ant-design/ant-design/pull/21387) [@orkhan-huseyn](https://github.com/orkhan-huseyn)
 - Typescript
-  - 🔷 添加 `MenuItemGroupProps` 导出。[#21356](https://github.com/ant-design/ant-design/pull/21356)
+  - 🔷 Menu 添加 `MenuItemGroupProps` 导出。[#21356](https://github.com/ant-design/ant-design/pull/21356)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md]:
```suggestion
  - 🔷 Menu 导出 `MenuItemGroupProps`。[#21356](https://github.com/ant-design/ant-design/pull/21356)
```

**Comment by yoyo837** [comment]
🇦🇿 这个看上去是字母还是国旗 emoji？

**Comment by chenshuai2144** [comment]
windows 下面就是az ，mac 上面是阿塞拜疆的国旗

**Review by afc163 — APPROVED** [review]

**Pull request merged**